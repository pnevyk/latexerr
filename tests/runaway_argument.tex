\documentclass{article}

\begin{document}
\date{April 2018 \maketitle

\date{April 2018 \maketitle
\end{document}
