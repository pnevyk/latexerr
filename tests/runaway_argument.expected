File: runaway_argument.tex

Error on line 4: Command \date was not properly ended with curly brace.
Error at the end: Command \date was not properly ended with curly brace.
